app/managers.py

```diff
@@ -0,0 +1,41 @@
+from models import Actor
+
+import sqlite3
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.sqlite")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} "
+            f"(first_name, last_name) VALUES (?, ?)",
+            (first_name, last_name,)
+        )
+        self._connection.commit()
+
+    def all(self) -> list:
+        actors_format_cursor = self._connection.execute(
+            f"SELECT * FROM {self.table_name}"
+        )
+        return [Actor(*row) for row in actors_format_cursor]
+
+    def update(self,
+               id_to_update: int,
+               new_first_name: str,
+               new_last_name: str) -> None:
+        self._connection.execute(
+            f"UPDATE {self.table_name} SET (first_name, last_name)"
```

Comment:
The SQL syntax in the `UPDATE` statement is incorrect. The correct syntax should be `SET first_name = ?, last_name = ?` instead of `SET (first_name, last_name) = (?, ?)`. This will ensure that the update operation works as intended.